mslib/msui/mpl_qtwidget.py

```diff
@@ -488,6 +488,8 @@ def __init__(self, model=None, settings=None, numlabels=None):
         self.settings_dict = {"vertical_extent": (1050, 180),
                               "vertical_axis": "pressure",
                               "secondary_axis": "no secondary axis",
+                              "plot_title_size": "default",
```

Comment:
we maybe should use here the config_loader and add the default value to the configuration. By this anyone can use his sizes from the mss_settings.json

https://github.com/Open-MSS/MSS/blob/develop/mslib/msui/__init__.py

We could add a dictionary here and start with 
```
topview = {"plot_title_size": 10,
                 "axes_label_size": 10}

sideview = {"plot_title_size": 10,
                 "axes_label_size": 10}

```

the config_loader can access this by 

topview_settings=config_loader(dataset="topview")

Reply:
> we maybe should use here the config_loader and add the default value to the configuration. By this anyone can use his sizes from the mss_settings.json
> 
> https://github.com/Open-MSS/MSS/blob/develop/mslib/msui/__init__.py
> 
> We could add a dictionary here and start with
> 
> ```
> topview = {"plot_title_size": 10,
>                  "axes_label_size": 10}
> 
> sideview = {"plot_title_size": 10,
>                  "axes_label_size": 10}
> ```
> 
> the config_loader can access this by
> 
> topview_settings=config_loader(dataset="topview")

Yes. I will look into it if I can do it.